### ISSUE TITLE
 vtkvmtkSurfMeshWrapper missing?

### ISSUE THREAD

**Issue by mv740** [vmtk/vmtk#182]
I was trying to use vtkvmtkSurfMeshWrapper but it wasn't included in the vtkvmtkMisc module. 
I tried to add it to the cmakelist but during compilation, I got an error "cannot open SurfaceMesher.h"

Is this file from a third party library?

**Comment by lantiga** [comment]
Yes, it is an old surface re-mesher from a colleague during my post-doc, it is not distributed anymore.
You can rely on vmtksurfaceremeshing, which is based on internal vmtk code.

We'll remove references to this class.

Thanks!
